components/shooter.py

```diff
@@ -165,6 +167,9 @@ def set_range(self, range: float) -> None:
             )
         )
 
+    def set_stop(self) -> None:
```

Comment:
I think `stop` is a verb here

```suggestion
    def stop(self) -> None:
```